### ISSUE TITLE
ExpandedType Package - Error when you have comments in inline type

### ISSUE THREAD

**Issue by arieffikhrie** [break-stuff/cem-tools#19]
I got error when analyze the code 
```bash
Error: 

[expand-types-plugin]: Looks like you've hit an error in third party plugin: expand-types-plugin. Please try to create a minimal reproduction and inform the author of the expand-types-plugin plugin.

 SyntaxError: Invalid regular expression: //** The name. */
    name?/: Nothing to repeat
    at new RegExp (<anonymous>)
    at /node_modules/cem-plugin-expanded-types/dist/index.js:56:7
    at Array.forEach (<anonymous>)
    at getObjectTypes (/node_modules/cem-plugin-expanded-types/dist/index.js:54:9)
    at getExpandedType (/node_modules/cem-plugin-expanded-types/dist/index.js:42:63)
    at /node_modules/cem-plugin-expanded-types/dist/index.js:46:30
    at Array.map (<anonymous>)
    at getUnionTypes (/node_modules/cem-plugin-expanded-types/dist/index.js:46:16)
    at getExpandedType (/node_modules/cem-plugin-expanded-types/dist/index.js:40:12)
    at getTypeValue (/node_modules/cem-plugin-expanded-types/dist/index.js:149:12)

    at withErrorHandling (/node_modules/@custom-elements-manifest/analyzer/src/utils/index.js:67:11)
    at /node_modules/@custom-elements-manifest/analyzer/src/create.js:104:7
    at Array.forEach (<anonymous>)
    at visitNode (/node_modules/@custom-elements-manifest/analyzer/src/create.js:103:19)
    at visitNodes (/Users/arfmy/Projects/cem-tools-test/node_modules/typescript/lib/typescript.js:29004:30)
    at Object.forEachChild (/Users/arfmy/Projects/cem-tools-test/node_modules/typescript/lib/typescript.js:29243:24)
    at visitNode (/node_modules/@custom-elements-manifest/analyzer/src/create.js:109:8)
    at analyze (/node_modules/@custom-elements-manifest/analyzer/src/create.js:100:3)
    at /node_modules/@custom-elements-manifest/analyzer/src/create.js:51:5
    at Array.forEach (<anonymous>)
```

This is the code being analyze.

```ts
import { html, LitElement } from 'lit';
import { customElement, state } from 'lit/decorators.js';

/**
 * The test element.
 */
@customElement('my-test')
export class Test extends LitElement {

  /** The data. */
  @state() data?: {
    /** The name. */
    name?: string,
    /** The type. */
    type?: string,
    /** The value. */
    value?: number,
  };

  render() {
    return html`<div>${JSON.stringify(this.data)}</div>
    </div>`
  }
}

```

It is because of i put comment on the Type declaration, if i remove the comments in the `data` type, it would working properly.
If i move the inline type as it own type, it also working fine.

**Comment by break-stuff** [comment]
Great catch! Let me take a look and see if I can find a solution. Thank you!

**Comment by break-stuff** [comment]
It looks like this is working for me. You may need to update your decorator. I also recommend defining your types and exporting them, so they can be used in the consumer's app.

```ts
import { html, LitElement } from 'lit';
import { customElement, property } from 'lit/decorators.js';

export type DataSet = {
    /** The name. */
    name?: string,
    /** The type. */
    type?: string,
    /** The value. */
    value?: number,
  };

/**
 * The test element.
 */
@customElement('my-test')
export class Test extends LitElement {

  /** The data. */
  @property({ attribute: false }) data?: DataSet;

  render() {
    return html`<div>${JSON.stringify(this.data)}</div>
    </div>`
  }
}
```

**Comment by arieffikhrie** [comment]
Right now, on my side, it would be ok for me to delete the comments in the type anyway.

But since I was adding this plugin to CEM on existing codebase, i would hope it would work out of the box without me debugging what is producing the error (In my case, i need to check the component code 1 by 1 to get the minimal error production). I think this would be the same case on new consumer of this plugin and might consider this plugin broken and will avoid to use this good plugin.

**Comment by break-stuff** [comment]
I was able to find the issue and deployed the fix in v1.1.4. Let me know if you run into any other issues.

**Comment by break-stuff** [comment]
I'm closing this for now. Let me know if anything else comes up.
